File: sdkwork-backend-sdk-go/types/plus_user_card_form.go
package types

// User-card binding creation form
type PlusUserCardForm struct {
	UserId int `json:"userId"`
	CardId int `json:"cardId"`
	CardCode string `json:"cardCode"`
	AcquireTime string `json:"acquireTime"`
	ActivateTime string `json:"activateTime"`
	CancelTime string `json:"cancelTime"`
	Points int `json:"points"`
	Balance int `json:"balance"`
}

File: sdkwork-backend-sdk-typescript/src/types/plus-user-card-form.ts
/** User-card binding creation form */
export interface PlusUserCardForm {
  /** User ID (reference to plus_user table) */
  userId: number;
  /** Card ID (reference to plus_card table) */
  cardId: number;
  /** User membership card number/Code */
  cardCode: string;
  /** Acquisition time */
  acquireTime: string;
  /** Activation time */
  activateTime?: string;
  /** Cancellation/expiration time */
  cancelTime?: string;
  /** Current points balance */
  points?: number;
  /** Current stored value balance */
  balance?: number;
}
